### ISSUE TITLE
Don't warn when ruby is found as a depend and is not going to be packaged

### ISSUE THREAD

**Issue by maxice8** [Cogitri/tmplgen#20]
```
$ tmplgen -t gem rb-readline
[WARN  libtmplgen::helpers] ruby-rb-readline of package description is longer than 80 characters, please cut as you see fit!
[WARN  libtmplgen::tmplwriter] Gem ruby is part of ruby, won't write a template for it!
[WARN  libtmplgen::tmplwriter] Won't write template for built-in package ruby
```

That message is printed out literally every time, i don't think we need to tell our users constantly about that.

**Comment by Cogitri** [comment]
Huh, this should be fixed since 6b3530b34e866b0ce25d63daa1c5e25df8e2f7a3 already. I'll look into it

**Comment by Cogitri** [comment]
Just tested, seems to already be fixed by this. You use tmplgen v1.2.0, right?

**Comment by Cogitri** [comment]
Derp, nevermind, that warning is also thrown when walking down the dependency tree, missed that.

**Comment by Cogitri** [comment]
Fixed in 1373138d5737d15a25bcfa77851e89e92f500a0e
